functions/src/index.ts

```diff
@@ -125,6 +125,10 @@ async function processWrite(
   const payload = change.after.data() as QueuePayload;
   logInfo("processing update");
 
+  if (!payload.delivery) {
+    logInfo("ignoring empty delivery update");
```

Comment:
Do you need to add a `return null;` to ignore it?

Reply:
Yes! Good catch